gui-libs/vte/vte-0.76.3.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit gnome.org meson python-any-r1 vala xdg
+
+DESCRIPTION="Library providing a virtual terminal emulator widget"
+HOMEPAGE="https://wiki.gnome.org/Apps/Terminal/VTE"
```

Comment:
Maybe replace by:
https://gitlab.gnome.org/GNOME/vte

Reply:
Done in https://github.com/gentoo/gentoo/pull/34844/commits/4823224d61b12fd97a25efcf977885dc04d7904f